orchestration/dagster_orchestration/hca_orchestration/tests/solids/load_hca/test_load_table.py

```diff
@@ -0,0 +1,110 @@
+from dagster import SolidExecutionResult, execute_solid
+from dagster_utils.contrib.data_repo.typing import JobId
+
+from hca_orchestration.pipelines.load_hca import test_mode
+from hca_orchestration.solids.load_hca.load_table import check_has_data, check_has_outdated, clear_outdated, \
+    load_table, start_load
+from hca_orchestration.support.typing import HcaScratchDatasetName, MetadataType, MetadataTypeFanoutResult
+
+
+run_config = {
+                "resources": {
+                    "scratch_config": {
+                        "config": {
+                            "scratch_bucket_name": "bucket_name",
+                            "scratch_prefix_name": "prefix_name",
+                            "scratch_bq_project": "bq_project",
+                            "scratch_dataset_prefix": "dataset_prefix",
+                            "scratch_table_expiration_ms": 86400000
+                        }
+                    },
+                    "target_hca_dataset": {
+                        "config": {
+                            "dataset_name": "dataset_name",
+                            "dataset_id": "dataset_id",
+                            "project_id": "project_id",
+                            "billing_profile_id": "billing_profile_id"
+                        }
+                    }
+                }
+            }
+metadata_fanout_result = MetadataTypeFanoutResult(
+    scratch_dataset_name=HcaScratchDatasetName("dataset"),
+    metadata_type=MetadataType("metadata"),
+    path="path"
+)
+
+
+def test_load_table():
```

Comment:
🔧 I see you're mixing pytest style bare function definitions here and `unittest.TestCase` subclasses [here](orchestration/dagster_orchestration/hca_orchestration/tests/solids/load_hca/test_ingest_metadata_type.py). Let's standardize on pytest bare function definitions going forward and not mix the two.